fabric/src/main/java/whocraft/tardis_refined/registry/fabric/DeferredRegistryImpl.java

```diff
@@ -29,7 +30,7 @@ public static <T> DeferredRegistry<T> create(String modid, ResourceKey<? extends
 
         public Impl(String modid, ResourceKey<? extends Registry<T>> resourceKey) {
             this.modid = modid;
-            this.registry = (Registry<T>) MoreObjects.firstNonNull(Registry.REGISTRY.get(resourceKey.location()), BuiltinRegistries.REGISTRY.get(resourceKey.location()));
+            this.registry = (Registry<T>) BuiltInRegistries.REGISTRY.get(resourceKey.location());
```

Comment:
@Suff99 Since the registry implementation was mostly your area, do you think this approach is alright?

Reply:
This is correct

Reply:
I think that was an accident. Might have been testing datagen and accidentally did something over here, reverting.

Reply:
Oh wait, it was because there was no Registries.REGISTRY anymore, there was nothing to compare.